packages/office-ui-fabric-react/src/utilities/color/index.ts

```diff
@@ -1,2 +1,24 @@
 export * from './colors';
```

Comment:
hm, is this line necessary anymore? not quite sure how these files work...

Reply:
I think since you make `colors` re-export all the new files, you could just use this line instead of all the new lines. (Though ideally colors.ts should be removed in a future version since it's redundant now.)